packages/webapp/webapp_tests.js

```diff
@@ -8,6 +8,24 @@ var hash = crypto.createHash('sha1');
 hash.update(additionalScript);
 var additionalScriptPathname = hash.digest('hex') + ".js";
 
+// convert a stream to a string via promise
+function toString(stream) {
+  return new Promise((success, fail) => {
+    var string = ''
+    stream.on('data', function(data) {
+      string += data.toString();
+    });
```

Comment:
I think it might be better if we appended the `data` buffers to an array and then used `Buffer.concat(chunks).toString("utf8")` at the end, since a chunk boundary could theoretically split a multi-byte unicode character.

Reply:
[Example](https://github.com/meteor/meteor/blob/70ffce52fc36e91e188cc1d10a67a63f9969f5dd/tools/utils/http-helpers.js#L19) of other code that we use for a similar purpose.